### PR TITLE
add mini deb conference 

### PR DESCRIPTION
**Pull request by championshuttler**
add mini deb conference  :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,7 +21,9 @@ If you would like to be notified of future conferences that would be added to th
...
 | [RootConf](https://rootconf.in/2017/) | 11-12 May | MLR Convention Centre, Bengaluru | Rootconf is India’s principal conference where systems and operations engineers share real world knowledge about building resilient and scalable systems. |
+| [Amazon AWS Summit](https://aws.amazon.com/summits/new_delhi/) | 12 May | JW Marriott Hotel Aerocity New Delhi | The AWS Summit brings together the cloud computing community to connect, collaborate and learn about AWS. In this action-packed, one-day event, you can choose from over 25 breakout sessions covering a broad range of topics and technical depth. Attend introductory sessions on new and existing AWS services or participate in sessions that go deep into architecture and code. All sessions will be delivered by subject matter experts from AWS, Partners or expert customers who can share their real world experiences and lessons learned.|
```
Comment (nikhita) [inline README.md]: Can you limit the description till just the first sentence? I think it conveys enough information.

I don't want the table to bloat up with huge descriptions. :smile: (though I agree I have some cleaning up to do wrt other conferences with big descriptions)

File: README.md [hunk 2]
```diff
@@ -21,7 +21,9 @@ If you would like to be notified of future conferences that would be added to th
 | [LaravelLive India](https://laravellive.in/) | 19 Mar | New Delhi | A PHP Laravel Conference. |
 | [Banglore Container Conference](http://www.containerconf.in/) | 7 Apr | Park Plaza (5 star), ORR, Bengaluru | Bengaluru Container Conference 2017 (BCC ’17) is the first conference on container technologies in India. The main focus of this conference is “Containers in Production”. |
 | [GIDS](http://www.developermarch.com/developersummit/) | 25-28 April | J. N. Tata Auditorium, Bengaluru | With over 40000 attendees benefiting from ten game changing editions, GIDS is the gold standard for India's software professional ecosystem.  |
+| [Oracle Code New Delhi](https://developer.oracle.com/code/newdelhi) | 09-10 May | Pragati Maidan New Delhi | Oracle Code is a free event for developers to learn about the latest developer technologies, practices, and trends.Learn from technical experts, industry leaders, and other developers in keynotes, sessions, and hands-on labs. Experience cloud development technology in the Code Lounge with workshops and other live, interactive experiences and demos. |
```
Comment (nikhita) [inline README.md]: Just the first sentence, please.

File: README.md [hunk 3]
```diff
@@ -31,6 +33,7 @@ If you would like to be notified of future conferences that would be added to th
...
 | [Eclipse Summit India](https://eclipsesummit.in/) | 29 July | Infosys Campus, Electronic City, Bengaluru | This is a great opportunity for the Eclipse fraternity in India to learn, explore, share, and collaborate on the latest ideas and information about Eclipse ecosystem and its community of projects with the core contributors, adopters, extenders, service providers, consumers and business and research organizations. |
 | [JS Channel](http://2017.jschannel.com/) | 28-29 July | JW Marriott Hotel, Bengaluru | JS Channel is a unique platform to hear best of the bests speakers in JS Community along with learning from expert JS trainers in workshop/live-coding sessions. |
+| [Oracle Code Bengaluru ](https://developer.oracle.com/code/bangalore) | 10 August | No 1, Residency Road Bengaluru, Karnataka | Oracle Code is a free event for developers to learn about the latest developer technologies, practices, and trends.Learn from technical experts, industry leaders, and other developers in keynotes, sessions, and hands-on labs. Experience cloud development technology in the Code Lounge with workshops and other live, interactive experiences and demos.|
```
Comment (nikhita) [inline README.md]: Just the first sentence, please.

**Comment by nikhita** [comment]
Oh btw, it works fine for this PR but I think you need a rebase for your branch. ;)

**Comment by championshuttler** [comment]
Ok Nikhita. Editing all of them and changing to one line sentence only :). is it ok?

**Comment by championshuttler** [comment]
And can you please elaborate this branch stuff i am confused here :(

**Comment by nikhita** [comment]
> Ok Nikhita. Editing all of them and changing to one line sentence only :). is it ok?

It's perfect! :tada: 

> And can you please elaborate this branch stuff i am confused here :(

The master branch of your fork - https://github.com/championshuttler/tech-conferences-india - is 6 commits ahead and 4 commits behind the original repo. So you'll need to update it later.

Don't worry about updating for this PR since even without the update, it will not cause conflicts. I also noticed that you sent the PR through your master branch. You probably want to create a new branch and _then_ make commits. Just do this after the PR is merged:

```bash
# add the original repo's url as upstream
$ git remote add upstream https://github.com/nikhita/tech-conferences-india.git

# make sure you are on master branch
$ git checkout master

# update the master branch to match the original repo
# note: use reset --hard because currently the changes are on your master branch
$ git reset --hard upstream/master
```

When adding any other changes:

```bash
# create a new branch
$ git checkout -b new-branch-name

# update this branch with respect to the main repo
$ git rebase upstream/master

# do the changes and commit them

# push the changes
$ git push
```

Sorry for the elaborate explanation if you already know this. If you don't, that's totally ok too. Feel free to ask questions on this. :)

**Pull request closed without merging**